mms/tests/integration_tests/onnx_export_integ_test.py

```diff
@@ -37,43 +62,81 @@ def cleanup(tmpdir):
     shutil.rmtree(tmpdir)
 
 
-def setup_onnx_integ(tmpdir):
+def setup_onnx_integ(tmpdir, URL,port):
     # Start the mxnet model server for onnx integ tests.
     print("Starting MXNet Model Server for onnx integ test..")
 
     # Set argv parameters
     sys.argv = ['mxnet-model-server']
-    sys.argv.append("--models")
-    sys.argv.append("squeezenet={}/squeezenet.model".format(tmpdir))
+    sys.argv.append('--models')
+    for onnx_model in URL.keys():
+        sys.argv.append("{}={}/{}.model".format(onnx_model, tmpdir, onnx_model))
+    sys.argv.append('--port')
+    sys.argv.append(port)
     mxnet_model_server.start_serving()
 
 
-def create_model(tmpdir):
+def create_model(tmpdir,URL, onnx_source_model_zoo = True):
     # Download the files required for onnx integ tests.
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/squeezenet.onnx")
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/signature.json")
-    _download_file(tmpdir, "https://s3.amazonaws.com/model-server/models/onnx-squeezenet/synset.txt")
-    # Download input image.
-    _download_file(tmpdir, "https://upload.wikimedia.org/wikipedia/commons/8/8f/Cute-kittens-12929201-1600-1200.jpg")
-    # Export the model.
-    print("Exporting the mxnet model server model...")
-    subprocess.check_call(['mxnet-model-export', '--model-name', 'squeezenet', '--model-path', tmpdir], cwd=tmpdir)
+    download_dir = tmpdir + '/scratch'
+    try:
```

Comment:
some more print messages after each state will be helpful for debugging failed test cases.